### DISCUSSION TITLE
Changing the unloaded/unselected tab title color/opacity (and removing any effects)?

### DISCUSSION THREAD

**Discussion by orenatol** [black7375/Firefox-UI-Fix#807] [Q&A]
Hello,

(Mac, FF Dev 120.0beta, and the latest (from yesterday) code from Git)

I'm using the Photon Colors theme being the closest to the contrast I need between active tab and the rest.

But I find the text color of the unloaded/unselected tabs titles hard to read, being in shades of grey.
I believe this is not part of the theme, as I see the same issue with the builtin themes too.

<img width="378" alt="Screenshot 2023-10-26 at 16 34 55" src="https://github.com/black7375/Firefox-UI-Fix/assets/59396584/a41900c0-a472-4ee5-a3ac-ddf41b72d5f1">

Is there a way to change this color to white, for example?
And to remove any opacity modification/effects?

**Comment by black7375** [comment]
You can set `userChrome.tab.unloaded` to `false` at `<FIREFOX_PROFILE>/user.js` (not `<FIREFOX_PROFILE>/chrome/user.js`)
https://github.com/black7375/Firefox-UI-Fix/blob/d7b0e558abba98fa40c6d3719ab2464af9b24e83/user.js#L243

Then, restart firefox please.

**Reply by orenatol** [reply]
This fixed the unloaded (i.e. inactive) tabs, thank you!


But the remaining issue for me is the unselected tabs.

They are still in some shade of grey. Neither black (light themes):
<img width="296" alt="Screenshot 2023-10-27 at 18 12 25" src="https://github.com/black7375/Firefox-UI-Fix/assets/59396584/05590491-cb5d-4eec-9d6b-29a98fa1b338">



... nor white (dark themes):
<img width="190" alt="Capture2" src="https://github.com/black7375/Firefox-UI-Fix/assets/59396584/2bc3eec1-a72f-4cfd-aa04-6a78cf99306e">



Probably this means it's not theme-dependent.


This is what I'm after:

<img width="267" alt="Screenshot 2023-10-27 at 18 13 40" src="https://github.com/black7375/Firefox-UI-Fix/assets/59396584/7ca21d23-108d-466b-9182-04cc41568b73">

(Mac, MS Edge, light theme)


![Capture1](https://github.com/black7375/Firefox-UI-Fix/assets/59396584/cba5849b-d54d-4c89-92fc-d8d8fee6e6f8)

(Win10, FF 115.4 ESR, а very old version of photon fix, system theme)
